watertap/examples/flowsheets/RO_with_energy_recovery/multi_sweep.py

```diff
@@ -0,0 +1,112 @@
+    m.fs.feed.properties[0].flow_vol_phase["Liq"].fix()
+    m.fs.feed.properties[0].conc_mass_phase_comp["Liq", "NaCl"].fix()
+    RO.optimize_set_up(m)
+    m.fs.eq_minimum_water_flux.deactivate()
+    RO.solve(m)
+    RO.display_system(m)
+    RO.display_design(m)
+
+    # uncomment to create outputs
+
+    outputs["LCOW"] = m.fs.costing.LCOW
+    # outputs["RO Water Flux"] = m.fs.RO.flux_mass_phase_comp[0, 1, "Liq", "H2O"]
+    # outputs["RO Membrane Area"] = m.fs.RO.area
+    # outputs["RO Energy Consumption"] = m.fs.costing.specific_energy_consumption
+    # outputs["System Capital Cost"] = m.fs.costing.aggregate_capital_cost
+    # outputs["RO Capital Cost"] = m.fs.RO.costing.capital_cost
+    # outputs["Pump Capital Cost"] = m.fs.P1.costing.capital_cost
+    # outputs["ERD Capital Cost"] = m.fs.ERD.costing.capital_cost
+    # outputs["RO Operating Cost"] = m.fs.RO.costing.fixed_operating_cost
+    # outputs[
+    #     "MLC Operating Cost"
+    # ] = m.fs.costing.maintenance_labor_chemical_operating_cost
+    # outputs["Feed Flow Rate"] = m.fs.feed.properties[0].flow_vol_phase["Liq"]
+    # outputs["Permeate Flow Rate"] = m.fs.product.properties[0].flow_vol_phase["Liq"]
+    # outputs["Retentate Flow Rate"] = m.fs.disposal.properties[0].flow_vol_phase["Liq"]
+    # outputs["RO Operating Pressure"] = m.fs.RO.inlet.pressure[0]
+    # outputs["RO Permeate H2O Mass Flow"] = m.fs.RO.permeate.flow_mass_phase_comp[
+    #     0, "Liq", "H2O"
+    # ]
+    # outputs["RO Permeate Salt Mass Flow"] = m.fs.RO.permeate.flow_mass_phase_comp[
+    #     0, "Liq", "NaCl"
+    # ]
+    # outputs["RO Retentate H2O Mass Flow"] = m.fs.RO.retentate.flow_mass_phase_comp[
+    #     0, "Liq", "H2O"
+    # ]
+    # outputs["RO Retentate Salt Mass Flow"] = m.fs.RO.retentate.flow_mass_phase_comp[
+    #     0, "Liq", "NaCl"
+    # ]
+
+    return outputs, m
+
+
+def run_analysis(case_num=1, nx=5, interpolate_nan_outputs=True, output_filename=None):
+
+    if output_filename is None:
+        output_filename = "sensitivity_" + str(case_num) + ".csv"
+
+    # when from the command line
+    case_num = int(case_num)
+    interpolate_nan_outputs = bool(interpolate_nan_outputs)
+
+    outputs, m = set_up_sensitivity()
+
+    # choose parameter sweep from case structure
+    sweep_params = {}
+
+    if case_num == 1:
+        # Need to unfix mass recovery of water (or simply sweep across it instead of recovery_vol)
+        m.fs.RO.recovery_mass_phase_comp.unfix()
+
+        sweep_params["mass_concentration"] = PredeterminedFixedSample(
+            m.fs.feed.properties[0].conc_mass_phase_comp["Liq", "NaCl"],
+            [0.963, 1.927, 4.816],
+        )
+        sweep_params["volumetric_recovery"] = PredeterminedFixedSample(
+            m.fs.RO.recovery_vol_phase[0, "Liq"], [0.7, 0.8, 0.9]
+        )
+    else:
+        raise ValueError(f"{case_num} is not yet implemented")
+
+    global_results = parameter_sweep(
```

Comment:
I would recommend updating the function call here to look more like the one here
https://github.com/watertap-org/watertap/blob/2c69710913ee3f61ac77fa3b9ab4abfb526b4eb6/watertap/examples/flowsheets/RO_with_energy_recovery/monte_carlo_sampling_RO_ERD.py#L122-L143

Reply:
@MarcusHolly we are technically using the "old" way of doing things with parameter sweep. It'd be good for us to update to the latest formulation, following @k1nshuk's example.

Reply:
I've tried updating the multisweep file, but I can't figure out why it's failing to converge with these changes. I've also tried a file locally that is much closer to the `monte_carlo_sampling_RO_ERD.py`, but it runs into the same infeasible solution issue.

Reply:
We need to go through and clean up all the examples of the old API anyways.

I wouldn't worry about it if it's a bottleneck for this PR.

Reply:
I've reverted this change so that we can just get this PR merged in. Perhaps there should be an issue mentioning that the old API will be deprecated. Not sure how urgent it is to update all of these files...

Reply:
OK, we'll save this update for a separate PR that addresses most, if not all, sweep cases.